tensorboard/uploader/uploader.py

```diff
@@ -166,6 +227,10 @@ def delete_experiment(writer_client, experiment_id):
         raise
 
 
+class InvalidArgumentError(RuntimeError):
+    pass
```

Comment:
OK: We do have `tensorboard.errors.InvalidArgumentError` already, but
it’s reasonable to want to keep these local, and it’s not a problem to
change later if we change our minds.